pkg/config/app/kuma-cp/config.go

```diff
@@ -36,6 +36,8 @@ type Defaults struct {
 	SkipMeshCreation bool `json:"skipMeshCreation" envconfig:"kuma_defaults_skip_mesh_creation"`
 	// If true, it skips creating the default tenant resources
 	SkipTenantResources bool `json:"skipTenantResources" envconfig:"kuma_defaults_skip_tenant_resources"`
+	// If true, it creates the default routing (TrafficPermisson and TrafficRoute) resources for a new Mesh
```

Comment:
I think we should improve this description by providing more specific examples of when these policies are necessary and explaining why they are not created by default anymore.